_posts/2024-03-06-happy-birthday.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Happy birthday OpenTTD!
+author: orudge
+---
+
+Exactly 21 years ago, I received an ICQ message (look it up, kids) from a guy named Ludvig Strigeus (nicknamed Ludde).
+"Hello, you probably don't know me, but I've been working on a project to clone TTD for a while." he said, more or less.
+He sent me a copy of what he'd been working on, and it was very exciting indeed.
+It was a fully functional version of Transport Tycoon Deluxe, written in C.
+Admittedly incomplete at that time, it was still quite remarkable.
+Ludde didn't want to release this to the public yet, and wasn't entirely sure what he was going to do with it.
+(He'd been working on it since 30th June 2002, so I guess that is technically the birthday of what would become OpenTTD...)
+
+<!-- more -->
```

Comment:
I have been wondering if we shouldn't put this between line 7 and 8. This is fine too, so ignore this if you like. More wondering out loud :)

Reply:
It maybe does read better that way, I've swapped it round to try. :)